src/video-player.js

```diff
@@ -144,6 +144,9 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       if (source) {
         this.source(source);
       }
+      let maxTries = (options.videojsOptions.maxTries) ? options.videojsOptions.maxTries : 3;
```

Comment:
In JavaScript 0 value is false in conditions flow:
`0 ? 0 : 3`
`=> 3` 

That means that there's no way to set maxTries to 0.

Reply:
Also, perhaps a more elegant way to write the same would be:
`let maxTries = options.videojsOptions.maxTries || 3`